component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -1923,4 +1939,42 @@ public static String getTranslatedLabel(String fieldName, long siteId, Locale lo
             locale);
 
   }
+  private static Visibility[] convertVisibilities(List<String> visibilityNames) {
+    if (visibilityNames == null) {
+      return Visibility.values();
+    }
+    return visibilityNames.stream()
+            .map(visibilityName -> Visibility.valueOf(StringUtils.upperCase(visibilityName)))
+            .toList()
+            .toArray(new Visibility[0]);
+  }
+  private static List<UserNodeRestEntity> removeEmptyNavigations(List<UserNodeRestEntity> userNodeList) {
```

Comment:
```suggestion
  private static void removeGroupNodesWithoutPageChildNodes(List<UserNodeRestEntity> userNodeList) {
```
Add a comment to explain what this method does exactly.